api/routes.go

```diff
@@ -64,6 +64,7 @@ func (s *Server) RegisterRoutes() {
 	// Graphite endpoints
 	r.Combo("/render", cBody, withOrg, ready, bind(models.GraphiteRender{})).Get(s.renderMetrics).Post(s.renderMetrics)
 	r.Combo("/metrics/find", withOrg, ready, bind(models.GraphiteFind{})).Get(s.metricsFind).Post(s.metricsFind)
+	r.Combo("/metrics/expand", withOrg, ready, bind(models.GraphiteExpand{})).Get(s.metricsExpand).Post(s.metricsExpand)
```

Comment:
please update docs/http-api.md and docs/cloud/http-api.md

Reply:
I added docs in `docs/http-api.md`. 
I'm not sure that it makes sense to also document it in `docs/cloud/http-api.md` because I think that by adding it there we kind of like recommend to our users to use that endpoint, but in reality it's really mostly redundant to `/metrics/find` and we only need to support it to be able to claim "graphite compatibility". I think if a user isn't already looking for this specific endpoint because they know it from graphite, there's no need to mention it, it's just cluttering the docs.

Reply:
I think we should document it

Reply:
ok, i added docs for it